### ISSUE TITLE
"Error, no model files in submission"

### ISSUE THREAD

**Issue by three-bee** [mdenna-synaptics/codalab2022#11]
Last update of the .json file says that our model cannot be found in the zip file. 

However, according to the latest challange submission guidelines, it is strictly stated that all three .tflite model files (model.tflite, model_none.tflite and model_none_float.tflite, first of which is quantized and has the hardcoded input & output dimensions) must be put in ./TFLite directory. 

As far as I understand, current method searches for the .tflite file where .png files are located, which is not correct for this stage of the competition. If .json file is going to be updated, this issue needs to be fixed.

**Comment by Lumos-Leo** [comment]
Just include the model.tflite and readme.txt files to check the time cost. Maybe only the last upload should follow the guidelines? That's how I do it. XD

**Comment by mdenna-synaptics** [comment]
You're right, I didn't update the script yet, thanks for pointing this out.
This is fixed now so **please follow the guidelines**
(the model that will be tested on the board is  ./TFLite/model.tflite)
